### PR TITLE
fix: quicksilver consensus module [web-quicksilver]

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

Closes: [BDU-785](https://forbole.atlassian.net/browse/BDU-785)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] ran linting via `yarn lint`
- [x] wrote tests where necessary
- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] targeted the correct branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [x] added a changeset via [`yarn && yarn changeset`](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md)

[BDU-785]: https://forbole.atlassian.net/browse/BDU-785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/shared-utils/configs/networks.json [hunk 1]
```diff
@@ -0,0 +1,439 @@
+[
```
Comment (teamchong) [inline packages/shared-utils/configs/networks.json:1]: Do you think it's necessary to keep this file? Personally, I would suggest deleting it and only retaining https://github.com/forbole/big-dipper-networks/blob/main/networks.json. What is your opinion on this?

Comment (rachelhox) [reply packages/shared-utils/configs/networks.json:1]: yes I think we can remove this if there's no reference to this file, instead we can move this generated file to update the `big-dipper-networks` repo

**Review by teamchong — CHANGES_REQUESTED** [review]
As per our discussion, please delete the file in question, as it does not appear to have any references or relevant information.

**Review by teamchong — APPROVED** [review]
LGTM

**Pull request merged**